internal/pkg/v2/bootstrap/container/container.go

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package container
+
+import (
+	"github.com/edgexfoundry/edgex-go/internal/core/data/config"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/controller/errorconcept"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/infrastructure/interfaces"
+
+	"github.com/edgexfoundry/go-mod-bootstrap/bootstrap/container"
+	"github.com/edgexfoundry/go-mod-bootstrap/di"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/metadata"
+
+	"github.com/edgexfoundry/go-mod-messaging/messaging"
+)
+
+type Container struct {
```

Comment:
Not wild about the generic name `Container`. What is this `thing` represent? File name should also be more specific.

Reply:
Rename from `Container` to `ServiceInjector` as this struct is used to save all the service clients injected by `go-mod-bootstrap`. Also rename the file to `serviceinjector.go`.